.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Question 🤔
+about: Usage question or discussion about the Carbon Design System.
```

Comment:
@joshblack Again, curious if this is meant to be at the Carbon Design System level, or if we should get more specific.